src/vivarium_gates_nutrition_optimization/components/children.py

```diff
@@ -15,22 +16,23 @@
 )
 
 
-class NewChildren:
-    def __init__(self):
-        self.lbwsg = LBWSGDistribution()
-
-    @property
-    def name(self):
-        return "child_status"
+class NewChildren(Component):
+    ##############
+    # Properties #
+    ##############
 
     @property
-    def sub_components(self):
+    def sub_components(self) -> List[str]:
         return [self.lbwsg]
 
     @property
-    def columns_created(self):
+    def columns_created(self) -> List[str]:
```

Comment:
Got it. I see now that the Pregnancy component also claims to create these columns. This is actually a misuse of the `columns_created` property and would cause an error if the `NewChildren` component had an `on_initialize_simulants` method.

By defining this property, you are also making it so that the `NewChildren` component will create a population view for itself with these columns, even though it doesn't need one.

Reply:
removed